packages/carbon-web-components/src/components/accordion/accordion-skeleton.ts

```diff
@@ -91,19 +91,20 @@ class CDSAccordionSkeleton extends LitElement {
     return html`
       ${this.open
         ? html`
-            <li class="${classes}">
-              <span class="${prefix}--accordion__heading">
+            <li class="${classes}" part="item item-open">
```

Comment:
Need to document this and all the other `part` attributes with `@csspart` annotations for the class.